zeppelin-web/src/components/websocketEvents/websocketEvents.factory.js

```diff
@@ -20,6 +20,13 @@ angular.module('zeppelinWebApp').factory('websocketEvents',
   websocketCalls.ws = $websocket(baseUrlSrv.getWebsocketUrl());
   websocketCalls.ws.reconnectIfNotNormalClose = true;
 
+  var getQueryString = function ( field, url ) {
```

Comment:
build fails with `jscs` checkstyle because of empty spaces around here:
```
[INFO] Running "jscs:all" (jscs) task
[INFO] disallowSpacesInAnonymousFunctionExpression: Illegal space before opening round brace at src/components/websocketEvents/websocketEvents.factory.js :
[INFO]     21 |  websocketCalls.ws.reconnectIfNotNormalClose = true;
[INFO]     22 |
[INFO]     23 |  var getQueryString = function ( field, url ) {
[INFO] ---------------------------------------^
[INFO]     24 |    var href = url ? url : window.location.href;
[INFO]     25 |    var reg = new RegExp( '[?&]' + field + '=([^&#]*)', 'i' );
[INFO] disallowSpacesInFunctionExpression: Illegal space before opening round brace at src/components/websocketEvents/websocketEvents.factory.js :
[INFO]     21 |  websocketCalls.ws.reconnectIfNotNormalClose = true;
[INFO]     22 |
[INFO]     23 |  var getQueryString = function ( field, url ) {
[INFO] ---------------------------------------^
[INFO]     24 |    var href = url ? url : window.location.href;
[INFO]     25 |    var reg = new RegExp( '[?&]' + field + '=([^&#]*)', 'i' );
[INFO] disallowSpacesInsideParentheses: Illegal space after opening round bracket at src/components/websocketEvents/websocketEvents.factory.js :
[INFO]     21 |  websocketCalls.ws.reconnectIfNotNormalClose = true;
[INFO]     22 |
[INFO]     23 |  var getQueryString = function ( field, url ) {
[INFO] -----------------------------------------^
[INFO]     24 |    var href = url ? url : window.location.href;
[INFO]     25 |    var reg = new RegExp( '[?&]' + field + '=([^&#]*)', 'i' );
[INFO] disallowSpacesInsideParentheses: Illegal space before closing round bracket at src/components/websocketEvents/websocketEvents.factory.js :
[INFO]     21 |  websocketCalls.ws.reconnectIfNotNormalClose = true;
[INFO]     22 |
[INFO]     23 |  var getQueryString = function ( field, url ) {
[INFO] ----------------------------------------------------^
[INFO]     24 |    var href = url ? url : window.location.href;
[INFO]     25 |    var reg = new RegExp( '[?&]' + field + '=([^&#]*)', 'i' );
[INFO] disallowSpacesInsideParentheses: Illegal space after opening round bracket at src/components/websocketEvents/websocketEvents.factory.js :
[INFO]     23 |  var getQueryString = function ( field, url ) {
[INFO]     24 |    var href = url ? url : window.location.href;
[INFO]     25 |    var reg = new RegExp( '[?&]' + field + '=([^&#]*)', 'i' );
[INFO] ---------------------------------^
[INFO]     26 |    var string = reg.exec(href);
[INFO]     27 |    return string ? string[1] : null;
[INFO] disallowSpacesInsideParentheses: Illegal space before closing round bracket at src/components/websocketEvents/websocketEvents.factory.js :
[INFO]     23 |  var getQueryString = function ( field, url ) {
[INFO]     24 |    var href = url ? url : window.location.href;
[INFO]     25 |    var reg = new RegExp( '[?&]' + field + '=([^&#]*)', 'i' );
[INFO] -------------------------------------------------------------------^
[INFO]     26 |    var string = reg.exec(href);
[INFO]     27 |    return string ? string[1] : null;
[INFO] requireCurlyBraces: If statement without curly braces at src/components/websocketEvents/websocketEvents.factory.js :
[INFO]     42 |      data.roles = $rootScope.ticket.roles;
[INFO]     43 |      data.runAs = getQueryString('runAs');
[INFO]     44 |      if (!data.runAs) data.runAs = $rootScope.ticket.principal;
[INFO] --------------^
[INFO]     45 |    } else {
[INFO]     46 |      data.principal = '';
[INFO] >> 7 code style errors found!
```